src/devices/Mpu9250/FifoModes.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Iot.Device.Mpu9250
+{
+    /// <summary>
+    /// FIFO modes used to select which data from accelerometer, gyroscope and I2C slaves
```

Comment:
not sure what this comment means :smile:

Reply:
ok, will rewrite a bit ;-)